crates/bevy_ecs/macros/src/lib.rs

```diff
@@ -1,9 +1,10 @@
 extern crate proc_macro;
 
-use find_crate::{Dependencies, Manifest};
+use cargo_manifest::{DepsSet, Manifest};
```

Comment:
Can you move this change out into a new PR?

Reply:
If it's a more preferrable way than just adding `use crate as bevy_ecs;` to `event.rs`, then sure.